xclim/indices/_multivariate.py

```diff
@@ -1417,3 +1464,53 @@ def winter_rain_ratio(
     ratio = liquid_precip_ratio(pr, prsn, tas, freq=freq)
     winter = ratio.indexes["time"].month == 12
     return ratio.sel(time=winter)
+
+
+@declare_units(
+    snd="[length]", sfcWind="[speed]", snd_thresh="[length]", sfcWind_thresh="[speed]"
+)
+def blowing_snow(
+    snd: xarray.DataArray,
+    sfcWind: xarray.DataArray,
+    snd_thresh: str = "5 cm",
+    sfcWind_thresh: str = "15 km/h",
+    window: int = 3,
+    freq: str = "AS-JUL",
+) -> xarray.DataArray:
+    """
+    Days with blowing snow events
+
+    Number of days where both snowfall over the last days and daily wind speeds are above respective thresholds.
+
+    Parameters
+    ----------
+    snd : xarray.DataArray
```

Comment:
This calculation uses what seems like a complicated way to derive snowfall  (snd.diff() etc) ...  Could we not get it from prsn?

Reply:
Totally, that was also my first impulse, but that's how the CCCS wants it.